docs/Changelog.md

```diff
@@ -15,6 +15,7 @@
 
 1.6.1 (Not released yet)
 * [#3647](https://github.com/TouK/nussknacker/pull/3647) Fix for serving OpenApi definition and SwaggerUI for deployed RequestResponse scenarios in embedded mode
+* [#3657](https://github.com/TouK/nussknacker/pull/3657) Fix for json-schema additionalProperties validation
```

Comment:
please prepare also PR to branch `release/1.6`

Reply:
I did, it was merged recently :)